[CTOR-1738]-Refresh applications-webservers-nginx-serverstatus doc page (#4492)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-webservers-nginx-serverstatus.md

Reviewer: ```suggestion
and add the followings lines in 'server' ( (if not already configured):
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
Make sure you allow pollers to access this URL.
```

Reviewer: ```suggestion
You can now check the result by accessing this URL:
```

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **Nginx Server** connector through the
**Configuration > Connectors > Monitoring Connectors** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

The Monitoring Connector **Nginx Server** brings a host template:

* **App-Webserver-Nginx-ServerStatus-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Webserver-Nginx-ServerStatus-custom" label="App-Webserver-Nginx-ServerStatus-custom">

| Service Alias     | Service Template                       | Service Description        |
|:------------------|:---------------------------------------|:---------------------------|
| Nginx-Connections | App-Webserver-Nginx-Connections-custom | Check current connections  |

> | Nginx-Requests    | App-Webserver-Nginx-Requests-custom    | Check request informations |

Reviewer: ```suggestion
| Nginx-Requests    | App-Webserver-Nginx-Requests-custom    | Check request information |
```

> Warning: The following procedure is an example. Cannot be applied on all context.

Reviewer: ```suggestion
Warning: The following procedure is an example and cannot be applied in all contexts.
```
